fastlane/lanes/localization.rb

```diff
@@ -222,15 +222,16 @@ def generate_strings_file(gutenberg_path:, derived_data_path:)
   # Updates the `AppStoreStrings.po` files (WP+JP) with the latest content from the `release_notes.txt` files and the other text sources
   #
   # @option [String] version The current `x.y` version of the app. Optional. Used to derive the `release_notes_xxy` key to use in the `.po` file.
+  # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
   #
   desc 'Updates the AppStoreStrings.po file with the latest data'
   lane :update_appstore_strings do |options|
     ensure_git_status_clean
 
-    release_version = release_version_current
+    version = options.fetch(:version, release_version_current)
```

Comment:
I've added this line to make it easier to test and run the lane independently.
The `version` parameter was already being mentioned in the documentation for the lane and it is also used by `update_wordpress_appstore_strings` / `update_jetpack_appstore_strings`.

Reply:
Did you consider updating the lane to use the named block parameters?

Reply:
Well reminded, yes... though it would require more changes (still, probably not much) in other places as we have `update_wordpress_appstore_strings` / `update_jetpack_appstore_strings` also using `options` as parameter. For now I'll to keep this fix with only the required changes and will tackle it in another PR.